SOURCES/zfs-2.1.15-break-multipathd-dependency-loop.patch

```diff
@@ -0,0 +1,13 @@
+diff -ru zfs-2.1.15.orig/etc/systemd/system/zfs-import-cache.service.in zfs-2.1.15/etc/systemd/system/zfs-import-cache.service.in
+--- zfs-2.1.15.orig/etc/systemd/system/zfs-import-cache.service.in	2024-02-27 00:04:26.084684081 +0100
++++ zfs-2.1.15/etc/systemd/system/zfs-import-cache.service.in	2024-05-03 16:15:44.434000000 +0200
+@@ -5,7 +5,8 @@
+ Requires=systemd-udev-settle.service
+ After=systemd-udev-settle.service
+ After=cryptsetup.target
+-After=multipathd.service
++#XCP-ng: break dependency loop, no ZFS over mpath?
```

Comment:
What is the loop dependency here? Are we sure that in any scenarios multipathd.service will be loaded after installing zfs? But in fact we don't need if currently as we don't use multipath... So I think that it is ok.

Reply:
We allow users to use it, as documented by XS:
* https://docs.xenserver.com/en-us/citrix-hypervisor/storage/multipath.html
* https://docs.xenserver.com/en-us/citrix-hypervisor/install/san-boot

I'll document the cycle better in the commit message.

Reply:
done